src/NRedisStack.Core/Bloom/BloomCommands.cs

```diff
@@ -5,30 +5,32 @@ namespace NRedisStack.Core
 
     public class BloomCommands
     {
+        //TODO: Check all the returned value of the commands and change if needed (Can add more parse methods)
         IDatabase _db;
         public BloomCommands(IDatabase db)
         {
             _db = db;
         }
 
-        public RedisResult Add(RedisKey key, string item)
+        public bool Add(RedisKey key, RedisValue item)
         {
-            return _db.Execute(BF.ADD, key, item);
+            return _db.Execute(BF.ADD, key, item).ToString() == "1";
         }
 
-        public bool Exists(RedisKey key, string item)
+        public bool Exists(RedisKey key, RedisValue item)
         {
             return _db.Execute(BF.EXISTS, key, item).ToString() == "1";
```

Comment:
What happens when this fails and an exception is raised (if the module isn't present, for example).

Reply:
It will return false.
What would you expect to happen?